packages/react-core/src/components/Pagination/Pagination.tsx

```diff
@@ -80,8 +80,10 @@ export interface PaginationProps extends React.HTMLProps<HTMLDivElement>, OUIAPr
   isDisabled?: boolean;
   /** Flag indicating if pagination is compact */
   isCompact?: boolean;
-  /** Flag indicating if pagination is not sticky */
+  /** Flag indicating if pagination is not sticky on mobile. Relevant when variant is bottom */
```

Comment:
It can also be used with top pagination if the pagination is on its own and not in a toolbar.